deployments/clowdapp.yml

```diff
@@ -199,6 +253,18 @@ parameters:
   value: 'true'
 - name: CLEANER_SCHEDULE
   value: "00 10 * * *"
+- name: DB_NAME
+  description: Database name used by the db-cleaner CronJob
+  value: postgres
```

Comment:
This one matches our database name in the [database definition](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/resources/terraform/resources/insights/stage/rds/rds-payload-tracker-stage-12.yml#L3).